x/wasm-storage/types/keys.go

```diff
@@ -0,0 +1,45 @@
+package types
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
```

Comment:
It may be a stylistic choice on my end, but I'm unsure if others would agree; however, I noticed we have some configured constants in many different files. I think it's easier to find/edit if they are usually in the same place, like a `consts.go` file, and then labeled appropriately in there. Food for thought, not a hard ask.

Reply:
That's a reasonable suggestion, but I'm just following the standards of Cosmos projects here. (putting all key-related stuff in `keys.go`)